module/VuFindSearch/src/VuFindSearch/Command/AbstractBase.php

```diff
@@ -106,6 +106,18 @@ public function getTargetIdentifier(): string
         return $this->backendId;
     }
 
+    /**
+     * Set target backend identifier.
+     *
+     * @param string $identifier New identifier
+     *
+     * @return void
+     */
+    public function setTargetIdentifier(string $identifier): void
```

Comment:
I don't see any calls to this; what is it used for?

Reply:
It's used Blender\Backend to retarget events.